cli/src/commands/get/users.rs

```diff
@@ -9,24 +9,54 @@ pub struct GetUsersArgs {
     #[structopt(short = "u", long = "user")]
     /// Use to retrieve a single user with the provided id
     user: Option<UserIdentifier>,
+
+    #[structopt(short = "o", long = "project")]
+    /// Filter users by a given project
+    project_name_filter: Option<ProjectName>,
+
+    #[structopt(short = "p", long = "permission")]
+    /// Filter users by a given project permission
+    project_permission_filter: Option<ProjectPermission>,
 }
 
 pub fn get(client: &Client, args: &GetUsersArgs, printer: &Printer) -> Result<()> {
-    let GetUsersArgs { user } = args;
-    match user {
+    let GetUsersArgs {
+        user,
+        project_name_filter,
+        project_permission_filter,
+    } = args;
+
+    if project_name_filter.is_none() && project_permission_filter.is_some() {
+        bail!("You cannot filter on `permission` without a `project`")
+    }
```

Comment:
I would argue you could implement this such that if you didn't pass in a `project`, you got back all users with that permission in any project.

But if that's not the behaviour you want, happy with this.

Reply:
Yeah good shout - will save that for a future PR. Thanks